licenseheader.txt

```diff
@@ -0,0 +1,13 @@
+/* 
```

Comment:
Hi Nils,
The licenseheader.txt file seems to be somewhat redundant with the existing LICENSE file. Can you please merge them into a single file? (you can keep the java comment-style as in your licenseheader.txt)